Templates/Android/generated/BaseMethodCollectionPage.java.tt

```diff
@@ -5,12 +5,26 @@
 <#host.TemplateName = BaseTypeCollectionPage(c);#>
 <#=writer.WriteHeader()#>
 <#=CreatePackageDef(host)#>
+<# 
+	var deltaPage = false;
+	if (TypeName(c).Length > 6 && TypeName(c).Substring(TypeName(c).Length-5) == "Delta") {
```

Comment:
Is there any chance that this could be true for something other than a delta page? Could something be named to make this true and not be a delta page?